packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/RadioInputs/components/RadioInput.js

```diff
@@ -25,7 +25,7 @@ export default function RadioInput (props) {
     questionId
   } = props
 
-  const backgroundColor = isChecked ? 'accent-2' : 'neutral-6'
+  const backgroundColor = isChecked ? 'accent-1' : 'neutral-6'
```

Comment:
🤦  I'm sorry I confused this PR with #2194! I'll try to sneak this in here though, if ok, or can include in a different PR.